### PR TITLE
Updated heartbeat for OSRAM bulb and Smartpower Outlet

### PR DESCRIPTION
**Pull request by larsfinander**
-OSRAM bulb config updated to make it check in every 5 min
-Update attribute heartbeat once after configuration is done
-Store timestamp each time data is received from device
-Removed polling from OSRAM bulb

### CODE REVIEW

File: devicetypes/smartthings/osram-lightify-led-tunable-white-60w.src/osram-lightify-led-tunable-white-60w.groovy [hunk 1]
```diff
@@ -146,22 +144,29 @@ def refresh() {
 
 def configure() {
     state.levelValue = 100
-    log.debug "Confuguring Reporting and Bindings."
+    log.debug "Configuring Reporting and Bindings."
```
Comment (workingmonk) [inline devicetypes/smartthings/osram-lightify-led-tunable-white-60w.src/osram-lightify-led-tunable-white-60w.groovy:147]: good to see the spelling mistake fixed :)

File: devicetypes/smartthings/smartpower-outlet.src/smartpower-outlet.groovy [hunk 2]
```diff
@@ -101,12 +102,16 @@ def meter() {
...
+	[
+			"st rattr 0x${device.deviceNetworkId} 1 0xB04 0x50B"
+	]
 }
 
 def configure() {
-	[
+	def configCmds = [
 			"zdo bind 0x${device.deviceNetworkId} 1 1 6 {${device.zigbeeId}} {}", "delay 200",
 			"zdo bind 0x${device.deviceNetworkId} 1 1 0xB04 {${device.zigbeeId}} {}"
```
Comment (workingmonk) [inline devicetypes/smartthings/smartpower-outlet.src/smartpower-outlet.groovy]: add a delay 200 to take care of the refresh piggybacking on the config commands

Comment (larsfinander) [inline devicetypes/smartthings/smartpower-outlet.src/smartpower-outlet.groovy]: DOne

**Comment by workingmonk** [comment]
:+1:

**Comment by bflorian** [comment]
:+1:

**Pull request merged**